pipedrive/client-php
# Fix activities due date mapping

## Problem
`addAnActivity` and `updateEditAnActivity` did not take into account `dueDate` as autocreated mapping was incorrect.

## How to test

```php
// test 1 - should create an activity with a due date
$query['subject'] = 'test 1';
$query['dueDate'] = '2021-03-28';

$client->getActivities()->addAnActivity($query);   

// test 2 - should update a due date for an existing activity 
$query['dueDate'] = '2021-03-29';
$query['id'] = '<an existing activity id>'

$client->getActivities()->updateEditAnActivity($query);   

// test 3 - should create an activity even if due date was not passed  
$query['subject'] = 'test 3';

$client->getActivities()->addAnActivity($query);   

```

Related PRs:
https://github.com/pipedrive/client-php/pull/28
https://github.com/pipedrive/client-php/pull/25
https://github.com/pipedrive/client-php/pull/22
https://github.com/pipedrive/client-php/pull/10
